src/main/scala/com/gu/mobile/content/notifications/lib/ContentAlertPayloadBuilder.scala

```diff
@@ -53,8 +53,8 @@ trait ContentAlertPayloadBuilder extends Logging {
 
   def buildPayLoad(content: Content, keyEvent: KeyEvent): ContentAlertPayload = {
     ContentAlertPayload(
-      title = Some(s"Liveblog update: ${keyEvent.title.getOrElse(content.webTitle)}"),
-      message = if (keyEvent.title.isDefined) Some(content.webTitle) else None,
+      title = Some(s"Liveblog update"),
+      message = if (keyEvent.title.isDefined) Some(s"${keyEvent.title.getOrElse(content.webTitle)}") else None,
```

Comment:
I think the if condition can be removed and always be `Some(s"${keyEvent.title.getOrElse(content.webTitle)}")`

Reply:
I tried this, but one of the tests relies on else being None ("create content alert for content & blogPost without title")

Reply:
You're right, then it should be ```keyEvent.title.map(_.webTitle)```